src/modules/DataCatalog/components/DataSource/ViewEdit/Edit/presenter.jsx

```diff
@@ -75,6 +78,10 @@ class Edit {
                   ? [
                     <div className='col-xs-6 col-md-6'>
                       <AttributeList />
+						{isManual ?
```

Comment:
`isManual` is always `false` for community edition since `CommonDataSourceDTO` does not contain `executionPolicy` field.